src/redux/mission/missionSlice.js

```diff
@@ -0,0 +1,48 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+const url = 'https://api.spacexdata.com/v3/missions';
+
+export const getMissions = createAsyncThunk('missions/fetch', async () => {
+  const response = await fetch(url);
+  const data = await response.json();
+  return data;
+});
```

Comment:
- [ ] Kindly ensure to filter the data from the missions API to only save the listed items per [the requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md#redux-fetch-data-and-update-redux-store).

  ![image](https://github.com/harshi0102/space-hub/assets/36542986/67098fd2-e4d0-4187-9d2a-aead271194b4)